presto-raptor/src/test/java/com/facebook/presto/raptor/storage/organization/TestCompactionSetCreator.java

```diff
@@ -103,22 +111,52 @@ public void testNonTemporalRowCountBasedOrganizationSet()
     public void testTemporalCompactionNoCompactionAcrossDays()
             throws Exception
     {
-        long day1 = Duration.ofDays(Duration.ofNanos(System.nanoTime()).toDays()).toMillis();
-        long day2 = Duration.ofDays(Duration.ofMillis(day1).toDays() + 1).toMillis();
-        long day3 = Duration.ofDays(Duration.ofMillis(day1).toDays() + 2).toMillis();
+        long today = Duration.ofNanos(System.nanoTime()).toDays();
```

Comment:
This doesn't make sense. `System.nanoTime()` is effectively an arbitrary 64-bit number and can be positive or negative. Why would we turn it into a number of days?

Reply:
I'm confused why we are changing this test at all.